### ISSUE TITLE
error in the text of the episode 'Before we start'

### ISSUE THREAD

**Issue by bienflorencia** [datacarpentry/R-ecology-lesson#852]
I found an error in the section [Organizing your working directory](https://datacarpentry.org/R-ecology-lesson/00-before-we-start.html#organizing-your-working-directory). 

The text says:

“We are going to keep the script in the root of our working directory because we are only going to use one file. Later, when you start create more complex projects, it might make sense to organize scripts in sub-directories.”

I think this could be changed to either 

- "Later, when you **create** more complex projects, (...)" or 
- "Later, when you **start creating** more complex projects, (...)".

**Comment by tobyhodges** [comment]
Thanks @bienflorencia for opening this issue. The lesson underwent a major update and reorganisation when https://github.com/datacarpentry/R-ecology-lesson/pull/887 was merged. As your suggestion relates to content in a version of the lesson before that update took place, I will close it. Please open a new issue if you have suggestions for how the new content could be improved.
